src/main/scala/it/pagopa/interop/backendforfrontend/service/impl/TenantProcessServiceImpl.scala

```diff
@@ -159,4 +149,25 @@ class TenantProcessServiceImpl(tenantprocessUrl: String, blockingEc: ExecutionCo
         )(BearerToken(bearerToken))
       invoker.invoke(request, s"Updating verified attribute $attributeId to $tenantId")
     }
+
+  override def addTenantMail(tenantId: UUID, seed: MailSeed)(implicit contexts: Seq[(String, String)]): Future[Unit] =
+    withHeaders[Unit] { (bearerToken, correlationId) =>
+      val request: ApiRequest[Unit] =
+        api.addTenantMail(xCorrelationId = correlationId, tenantId = tenantId, mailSeed = seed)(
+          BearerToken(bearerToken)
+        )
+      invoker.invoke(request, s"Add mail ${seed.address} to $tenantId")
```

Comment:
sensitive information
```suggestion
      invoker.invoke(request, s"Add mail to tenant $tenantId")
```